libs/api/__tests__/api/private/index.ts

```diff
@@ -959,6 +959,7 @@ export type MemberPlan = {
   id: Scalars['ID'];
   image?: Maybe<Image>;
   maxCount?: Maybe<Scalars['Int']>;
+  migrateToTargetPMid?: Maybe<Scalars['ID']>;
```

Comment:
Why is here just the ID field compared to  availablePaymentMethods where the PaymentMethode is passed?

Reply:
the migration property lives on the member plan directly and not on a n:m available payment method relations. that is how available payment methods should also be related some day.